plotter/main.go

```diff
@@ -49,22 +51,61 @@ var examples = []struct {
 }
 
 var formats = []string{
-	".eps",
-	".pdf",
-	".svg",
-	".png",
-	".tiff",
-	".jpg",
+	"eps",
+	"pdf",
+	"svg",
+	"png",
+	"tiff",
+	"jpg",
+}
+
+type canvasWriter interface {
```

Comment:
>It doesn't work with the dots in there. It doesn't seem like it would have worked in the old version either but I didn't check.

I'm confused; Save and and WriteTo take difference strings semantically. Save takes a file name destination while WriteTo takes a format.